ecommerce-app/src/component/CartItem.js

```diff
@@ -0,0 +1,32 @@
+// components/CartItem.js
+import React from "react";
+
+const CartItem = ({ item, quantity, onQuantityChange }) => {
+  return (
+    <div className="cart-item">
+      <div className="product-details">
+        <p>{item.name}</p>
+        <p>{item.description}</p>
+      </div>
+      <div className="quantity-dropdown">
+        <label htmlFor={`quantity-${item.id}`}>Quantity:</label>
+        <select
+          id={`quantity-${item.id}`}
+          value={quantity}
+          onChange={(e) => onQuantityChange(item.id, parseInt(e.target.value))}
+        >
+          {[1, 2, 3, 4, 5].map((qty) => (
```

Comment:
Quantity usually depends on product. Each product has a maximum supported quantity. This should be part of your product detail.